validator/src/main/java/com/amazon/aoc/validators/AbstractMetricsValidator.java

```diff
@@ -15,65 +15,50 @@
 
 package com.amazon.aoc.validators;
 
-import com.amazon.aoc.models.CloudWatchContext;
 import com.amazon.aoc.callers.ICaller;
 import com.amazon.aoc.exception.BaseException;
 import com.amazon.aoc.exception.ExceptionCode;
 import com.amazon.aoc.fileconfigs.FileConfig;
-import com.amazon.aoc.helpers.MustacheHelper;
 import com.amazon.aoc.helpers.RetryHelper;
 import com.amazon.aoc.models.Context;
 import com.amazon.aoc.models.ValidationConfig;
 import com.amazon.aoc.services.CloudWatchService;
 import com.amazonaws.services.cloudwatch.model.Dimension;
 import com.amazonaws.services.cloudwatch.model.Metric;
 import com.amazonaws.services.cloudwatch.model.MetricDataResult;
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.amazonaws.util.StringUtils;
 import lombok.extern.log4j.Log4j2;
 
-import java.nio.charset.StandardCharsets;
 import java.sql.Date;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
 /**
- * ContainerInsightMetricsValidator is diverged from CWMetricValidator for container insight
+ * AbstractMetricsValidator is diverged from CWMetricValidator for container insight
  * metrics validation. In this validator, we exclude specific dimension checking, e.g. OTelLib
  * and metrics rolling up.
  */
 // TODO: We hope to merge it with CWMetricValidator when the validation process is finalized.
 @Log4j2
-public class ContainerInsightMetricsValidator implements IValidator {
+public abstract class AbstractMetricsValidator implements IValidator {
```

Comment:
Should we call it `AbstractCWMetricsValidator` to make it clear it is only for cloudwatch metrics.

Reply:
Done